src/Algebra/Definitions/RawMagma.agda

```diff
@@ -1,4 +1,4 @@
-------------------------------------------------------------------------
```

Comment:
Please fix the 5 style guide violations in this file.

Reply:
I hope that I have found this and fixed (but not very sure).